src/core/public/chrome/ui/header/header.tsx

```diff
@@ -141,6 +185,36 @@ function findClosestAnchor(element: HTMLElement): HTMLAnchorElement | void {
   }
 }
 
+function getGroupIcon(groupName: AppCategory) {
+  switch (groupName) {
+    case AppCategory.management:
+      return 'managementApp';
+  }
+}
+
+function getGroupLabel(groupName: AppCategory) {
+  switch (groupName) {
+    case AppCategory.analyze:
+      return i18n.translate('core.ui.analyzeNavList.label', {
+        defaultMessage: 'Analyze',
+      });
+    case AppCategory.observability:
+      return i18n.translate('core.ui.observabilityNavList.label', {
+        defaultMessage: 'Observability',
+      });
+    case AppCategory.security:
+      return i18n.translate('core.ui.securityNavList.label', {
+        defaultMessage: 'Security',
+      });
+    case AppCategory.management:
+      return i18n.translate('core.ui.managementNavList.label', {
+        defaultMessage: 'Management',
+      });
+    default:
+      return groupName;
+  }
```

Comment:
As explained in a previous comment, I think it would be better to have the application service expose an API to retrieve the list of registered categories (and their associated label and icon), which will be consumed by the chrome service, than hardcoding the mapping here. It seems cleaner in term of responsibilities and it would allow to add new categories more easily if we decide to allow this for 3rd parties at some point.